projects/plugins/jetpack/extensions/blocks/conversation/utils.js

```diff
@@ -8,3 +13,7 @@ export function getParticipantByLabel ( participants, participantLabel ) {
 	const part = participants.filter( ( { label } ) => ( label?.toLowerCase() === participantLabel?.toLowerCase() ) );
 	return part?.length ? part[ 0 ] : null;
 }
+
+export function cleanFormatStyle( html ) {
+	return getTextContent( create( { html } ) )?.trim();
```

Comment:
It seems strange that we need to convert the string into a `RichText` value to remove the HTML tags and turn it back into a string. But it's working 😂